docs/UserGuide.md

```diff
@@ -410,7 +438,7 @@ The image below show you what ModCraft should look like after executing the `inf
   <img src="images/ImagesForUG/info-after.png" alt="After executing info command" style="height:400px; margin:20px;">
 </div>
 <p style="text-align: center;">
-    The command `info CS3230` is executed. Information about the module `CS3230` is displayed.
+    The command info CS3230 is executed. Information about the module CS3230 is displayed.
```

Comment:
I think we can leave the back ticks. Just wrap this text in md tag

Reply:
Updated👍